pset-functions.js

```diff
@@ -31,6 +31,25 @@ console.log('Problem 1:')
 
 // Add your code below this line
 
+function blackJack(playerCardScore, dealerCardScore) {
+
+    if (playerCardScore>21 && dealerCardScore>21) {
+      result = 0
+  } else if (playerCardScore>21 || dealerCardScore>21) {
+      result = Math.min(playerCardScore, dealerCardScore)
+  }
+    else {
+      result = Math.max(playerCardScore, dealerCardScore)
+  }
+  return result
+}
+
+const a=19
+const b=21
+console.log("Test case a=19 and b=21.  Winner is:")
+console.log(blackJack(a,b))
+
```

Comment:
@AtleLohrmann Good job here creating a function and utilizing the conditionals to get to your solution. Nice job utilizing the Math library methods max and min it's a nice feature for this use case.
take a look at this alternate solution [here](https://github.com/jsr20190826/hw-02-functions-solution/blob/a4d789161886ff115657133c279062df8a4ca72d/pset-functions.js#L30)